fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -997,27 +1165,47 @@ mod tests {
         }
     }
     impl EngineStartState for TestPneumatic {
-        fn left_engine_state(&self) -> EngineState {
-            self.fadec.engine_state(1)
-        }
-        fn right_engine_state(&self) -> EngineState {
-            self.fadec.engine_state(2)
+        fn engine_state(&self, engine_number: usize) -> EngineState {
+            self.fadec.engine_state(engine_number)
         }
         fn engine_mode_selector(&self) -> EngineModeSelector {
             self.fadec.engine_mode_selector()
         }
     }
     impl PackFlowValveState for TestPneumatic {
-        fn pack_flow_valve_is_open(&self, pack_id: usize) -> bool {
-            self.packs[pack_id].pfv_open_amount() > Ratio::default()
+        fn pack_flow_valve_is_open(&self, fcv_id: usize) -> bool {
+            let id = {
+                if fcv_id == 1 || fcv_id == 2 {
+                    0
+                } else {
+                    1
+                }
+            };
```

Comment:
Just a nitpick as: can we make a simple fcv to pack id function as this is also used in pack_flow_valve_air_flow() ?

Reply:
Good point! I created a function now in `A380AirConditioning` so it can be used in the test and in the pneumatic module.